LeveGenerator.cs

```diff
@@ -38,6 +43,37 @@ public static void Generate(LeveDatabase db, ObservableCollection<Leve> currentO
             }
         }
 
+        private static string WriteLisbeth(LeveDatabase db, ObservableCollection<Leve> currentOrder) {
+            var lisbethOrder = $@"
+        <Lisbeth Json=""[";
+
+            foreach (var leve in currentOrder.OrderBy(i => i.Level))
+            {
+                var amount = (leve.Repeats > 0) ? leve.NumItems * (leve.Repeats + 1) : leve.NumItems;
```

Comment:
so it looks like we are doing all of the crafting at the start, what if someone is using this to level? 

Is it better to craft, then turn in, then craft etc ?